src/styles/blocks/header-block.scss

```diff
@@ -0,0 +1,64 @@
+header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  margin: 0;
+}
+
+.nav__list {
+  height: 60px;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
+  height: 100%;
+  position: relative;
+  text-transform: uppercase;
+}
+
+.nav__item:last-child {
+  margin: 0;
+}
+
+.logo-container {
+  height: 40px;
+  margin: 10px 0;
+}
+
+.nav__link {
+  height: 100%;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: $main-color-link;
```

Comment:
The variable `$main-color-link` is used here. Ensure that it is defined in your variables file or elsewhere in your project. If it's not defined, this will cause an error.